src/software/ai/hl/stp/tactic/goalie/goalie_tactic.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "shared/parameter/cpp_dynamic_parameters.h"
+#include "software/ai/hl/stp/tactic/goalie/goalie_fsm.h"
+#include "software/ai/hl/stp/tactic/tactic.h"
+//#include "software/ai/evaluation/enemy_threat.h"
```

Comment:
I tried running locally and I needed to shuffle things around to make it work
```suggestion
#include "software/ai/hl/stp/tactic/tactic.h"
#include "software/ai/hl/stp/tactic/goalie/goalie_fsm.h"

//#include "software/ai/evaluation/enemy_threat.h"
```
Also there's an extra commented out include

Reply:
removed